BootloaderCorePkg/Include/Library/BootloaderCoreLib.h

```diff
@@ -90,7 +91,7 @@ typedef struct {
   UINT32        Offset;
   UINT32        Size;
   UINT32        Reserved;
-  UINT8         Hash[32];
+  UINT8         Hash[HASH_DIGEST_MAX];

```

Comment:
MULTI_PAYLOAD_ENTRY is not used by anybody. Let us remove the structure.

Reply:
will do.